p2p/host/autorelay/autorelay_test.go

```diff
@@ -121,7 +119,7 @@ func TestSingleCandidate(t *testing.T) {
 	)
 	defer h.Close()
 
-	require.Eventually(t, func() bool { return numRelays(h) > 0 }, 3*time.Second, 100*time.Millisecond)
+	require.Eventually(t, func() bool { return numRelays(h) > 0 }, 10*time.Second, 100*time.Millisecond)
```

Comment:
Should be, but as a rule of thumb I just made these all 10 seconds since if that fails in CI you know for sure it isn't a timing issue. I think I saw some cases where 3s was not enough